lib/barcodes_list_view.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:barcode_widget/barcode_widget.dart';
+import 'package:flutter/material.dart';
+import 'barcode_item.dart';
+import 'barcode_detail_page.dart';
+
+class BarcodesListView extends StatelessWidget {
+  const BarcodesListView({
+    super.key,
+    required this.barcodes,
+  });
+
+  final List<BarcodeItem> barcodes;
+
+  @override
+  Widget build(BuildContext context) {
+    return ListView.builder(
+      itemCount: (barcodes.length / 2).ceil(),
+      itemBuilder: (context, index) {
+        final leftBarcode = barcodes[index * 2];
+        final rightBarcode = (index * 2 + 1 < barcodes.length)
+            ? barcodes[index * 2 + 1]
+            : null;
+
+        return Padding(
+          padding: const EdgeInsets.symmetric(vertical: 4.0),
+          child: Row(
+            children: [
+              Expanded(
```

Comment:
To powinno być wyniesione do oddzielnej funkcji tak jak było wcześniej w `BarcodesPage`